tests/nlu/base/test_evaluation.py

```diff
@@ -286,6 +291,51 @@ def test_run_cv_evaluation():
     assert len(entity_results.test["CRFEntityExtractor"]["F1-score"]) == n_folds
 
 
+def test_run_cv_evaluation_with_response_selector():
+    td = training_data.load_data("data/examples/rasa/demo-rasa.md")
```

Comment:
optional: Let's spell that out
```suggestion
    training_data = training_data.load_data("data/examples/rasa/demo-rasa.md")
```